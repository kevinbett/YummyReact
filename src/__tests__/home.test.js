import React from 'react';
import {shallow} from 'enzyme';
import {shallowToJson} from 'enzyme-to-json';

import Navigation from '../homepage/home';

describe('Navigation Component', () => {
    const wrapper = shallow(<Navigation />);
    it('renders without crashing', () => {
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
});