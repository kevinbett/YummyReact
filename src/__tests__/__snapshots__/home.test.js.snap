// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation Component renders without crashing 1`] = `
<Paper
  circle={false}
  className="footer"
  rounded={true}
  transitionEnabled={true}
  zDepth={1}
>
  <BottomNavigation
    selectedIndex={0}
  >
    <BottomNavigationItem
      icon={
        <FontIcon
          className="muidocs-icon-action-home"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        />
      }
      label="Home"
      onClick={[Function]}
    />
    <BottomNavigationItem
      icon={
        <FontIcon
          className="material-icons"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        />
      }
      label="Heroku"
      onClick={[Function]}
    />
    <BottomNavigationItem
      icon={
        <FontIcon
          className="material-icons"
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
        />
      }
      label="Github"
      onClick={[Function]}
    />
  </BottomNavigation>
</Paper>
`;
